### DISCUSSION TITLE
Show Clientname in title bar of chat and settings window

### DISCUSSION THREAD

**Discussion by wschell** [jamulussoftware/jamulus#1352] [UI Related]
I now often use several jamulus clients in parallel. But the chat and settings window have only "Chat" and "Settings" als title, so it would be helpful, to have the --clientname printed - like for the main windows.
Thx
Walter

**Comment by gilgongo** [comment]
Not sure I can think of a downside to this, other than simply implementation time. Yes, it's an edge case, but having anonymous child windows seems undesirable.

**Reply by dcorson-ticino-com** [reply]
Shouldn't this then also be added in the Connect and Profile window titles in that case?

**Reply by wschell** [reply]
Yes, sure. 
But it was not in my focus, because often the chat and settings window stay open- for me the others not.
